compiler/src/iree/compiler/Dialect/Flow/Transforms/OutlineDispatchRegions.cpp

```diff
@@ -238,8 +238,24 @@ summarizeDispatchWorkgroupsOp(DispatchWorkgroupsOp regionOp) {
           // No cost estimation implemented, skip.
         });
   });
-  if (!bestOp)
-    return "";
+
+  if (!bestOp) {
+    std::string ret = "";
```

Comment:
nit: match existing naming in the function
```suggestion
    std::string bestSummary = "";
```